util/src/main/java/org/datacommons/util/McfMutator.java

```diff
@@ -32,18 +32,18 @@
 
   public static Mcf.McfGraph mutate(Mcf.McfGraph.Builder graph, LogWrapper logCtx) {
     McfMutator m = new McfMutator();
-    m.graph = graph;
+    m.graph = Mcf.McfGraph.newBuilder();
```

Comment:
I don't follow this change, what was the problem?

I thought it slightly better that we set everything there is in graph first, otherwise we might need to copy over all non-nodes fields (right now type https://github.com/datacommonsorg/import/blob/master/util/src/main/proto/Mcf.proto#L53 but it can grow).

Reply:
ConcurrentModificationException got thrown when updating the nodesMap while iterating over it. I think this is because the nodesMap is an UnmodifiableMap: https://screenshot.googleplex.com/b24GVJLrkLPNsjp

Reply:
Oh ok. Then maybe we can store the node names in a list and loop over those?  Like in http://google3/datacommons/import/util/mcf_schema_util.cc?l=433-438&rcl=377096245

Reply:
done